WcaOnRails/app/models/registration.rb

```diff
@@ -105,12 +105,15 @@ def best_solve(event, type)
   end
 
   def entry_fee
-    competition.base_entry_fee
+    Money.new(
+      competition.base_entry_fee_lowest_denomination + competition_events.sum(:event_entry_fee_lowest_denomination),
```

Comment:
`.sum` just calls the `+` operator (see [docs](https://apidock.com/rails/Enumerable/sum)). The `Money` class definitely implements that.

Maybe it's that `competition_events` is some ActiveRecord relation type object rather than an `Enumerable`. Does `competition_events.to_a.sum(:event_entry_fee)` work?